src/account.rs

```diff
@@ -56,14 +57,25 @@ pub async fn account_prompt_internal(account_handle: AccountHandle) -> Result<bo
             if let Err(err) = match account_cli.command {
                 AccountCommand::NewAddress => new_address_command(&account_handle).await,
                 AccountCommand::Balance => balance_command(&account_handle).await,
+                AccountCommand::BurnNativeToken { token_id, amount } => {
+                    burn_native_token_command(&account_handle, token_id, amount).await
+                }
+                AccountCommand::BurnNft { nft_id } => burn_nft_command(&account_handle, nft_id).await,
                 AccountCommand::Claim => claim_command(&account_handle).await,
                 AccountCommand::Consolidate => consolidate_command(&account_handle).await,
+                AccountCommand::DestroyAlias { alias_id } => destroy_alias_command(&account_handle, alias_id).await,
+                AccountCommand::DestroyFoundry { foundry_id } => {
+                    destroy_foundry_command(&account_handle, foundry_id).await
+                }
                 AccountCommand::Exit => {
                     return Ok(true);
                 }
                 AccountCommand::Faucet { url, address } => faucet_command(&account_handle, url, address).await,
                 AccountCommand::Addresses => addresses_command(&account_handle).await,
                 AccountCommand::Transactions => transactions_command(&account_handle).await,
+                AccountCommand::MeltNativeToken { token_id, amount } => {
```

Comment:
I know it's not part of your PR but can you reorder them alphabetically please ? Looks like I forgot to do it when I renamed the variants